src/utilities/resource-list/types.ts

```diff
@@ -1,3 +1,9 @@
+import {CheckboxHandles} from '../../types';
+
 export type ResourceListSelectedItems = string[] | 'All';
 
+export type CheckableButtonNode = CheckboxHandles;
```

Comment:
Do we need this type?

Reply:
Not really I guess. I wasn't thinking it made sense just because of the naming, but I can just us the CheckboxHandles. I'll remove it since it brings up the question.